system-tests/test/deposit-redemption.test.ts

```diff
@@ -167,11 +226,18 @@ describe("System Test - Deposit and redemption", () => {
       // Since the reveal deposit logic does not perform SPV proof, we
       // can reveal the deposit transaction immediately without waiting
       // for confirmations.
-      await TBTC.revealDeposit(
-        depositUtxo,
-        deposit,
-        electrumClient,
-        depositorBridgeHandle
+      const rawDepositTransaction = await electrumClient.getRawTransaction(
+        depositUtxo.transactionHash
+      )
```

Comment:
Let's use `depositorSdk.bitcoinClient` instead of `electrumClient`.

Reply:
Done in 3c4fd74455625a9acd2952991aa6da3db517ad5a.